Xamarin.Forms.Platform.Android/CellAdapter.cs

```diff
@@ -257,6 +256,9 @@ bool HandleContextMode(AView view, int position)
 
 		void OnActionItemClickedImpl(IMenuItem item)
 		{
+			if (ActionModeContext == null)
```

Comment:
This seems to be some kind of race condition. When you tap an action, `OnActionItemClicked` is called. This invokes `_supportActionMode?.Finish()` which triggers `OnDestroyActionMode`. Now, inside this method, we are setting `ActionModeContext` to nul. By the time, the menu disappears from screen, we can tap on the action again in which case we end up with a crash because `ActionModeContext` is null.

Another solution is to do `item.SetEnabled(false);` inside `OnActionItemClicked`. I changed the PR to use this version instead. This will stop `OnActionItemClicked` from being called twice.